src/codecs/person/README.md

```diff
@@ -1 +1,145 @@
-# `person`
+---
+title: person
+authors: []
+---
+
+# `person`: a codec for plain text representation of humans
+
+The `person` codec is most often used for decoding a simple string of plain text representing a person e.g. in the YAML header of a R Markdown file, or the `package.json` file of a NPM Javascript package.
+
+## Decoding
+
+Decoding a string is done in two phases.
+
+### ORCID detection and use
+
+If the string contains an [ORCID](https://orcid.org), then the [`orcid`](../orcid) codec is used to fetch the metadata for that person and other data in the string is ignored. For example,
+
+```person import=josiah
+Miss Josiah Carbeeeerry (https://orcid.org/0000-0002-1825-0097)
+```
+
+is decoded to,
+
+```json export=josiah
+{
+  "givenNames": [
+    "Josiah"
+  ],
+  "familyNames": [
+    "Carberry"
+  ],
+  "type": "Person"
+}
+```
+
+The ORCID can appear anywhere in the string, optionally prefixed with `http://orcid.org/` or `ORCID`. For example, this raw ORCID,
+
+```person import=stephen
+0000-0002-9079-593X
+```
+
+is decoded to
+
+```json export=stephen
+{
+  "givenNames": [
+    "Stephen"
+  ],
+  "familyNames": [
+    "Hawking"
+  ],
+  "type": "Person"
+}
+```
+
+> ⚠️ The `orcid` codec is work in progress and when complete willl return more metadata than shown above.
+
+### Text parsing
+
+If no ORCID is detected, or if ORCID decoding fails (e.g. due to no network access) then the text string is parsed to extract:
```

Comment:
Not _specifically_ related to this but I wonder if it would be useful to run Encoda with an option to disable network functions. For this function it would fall back to string parsing. Other functions that require network would fail with an error.

Reasons for offering this: privacy for users, speed up codecs.

Reply:
That seems like a worthwhile feature, not least of all to allow for testing that Encoda is usable without a connection. Since we are using `nock` already, the easiest way to implement that may be to simply run `nock.disableNetConnect()` if the `--nonet` option is supplied. Only issue there is that it may interfere with `puppeteer` - although `puppeteer.launch({ pipe: true })` may provide a workaround.